roles/deploy/tasks/nfs.yml

```
@@ -0,0 +1,24 @@
+---
+
+- name: install nfs-utils
+  yum: name=nfs-utils state=latest
+  vars:
+    nfsmounts:
+      - {path: "/home/scud/downloadchd/email", src: "ipo-file-svm-lif-be1.internal.ch:chd_image"}
+      - {path: "/home/scud/downloadceu/email", src: "ipo-file-svm-lif-be1.internal.ch:chcc_email"}
+      - {path: "/home/scud/downloadwck/email", src: "ipo-file-svm-lif-be1.internal.ch:wck_image"}
+      - {path: "/home/scud/downloadceu/post", src: "ipo-file-svm-lif-be1.internal.ch:chcc_post"}
+      - {path: "/data1/apps/scud", src: "ipo-file-svm-lif-be1.internal.ch:scanning"}
+      - {path: "/data1/home/scanning", src: "ipo-file-svm-lif-be1.internal.ch:tiffexport"}
+      - {path: "/data1/image/scud", src: "ipo-file-svm-lif-be1.internal.ch:tiffexport"}
+
+- name: mount the nfsshare in client side
+  mount:
+    fstype: nfs
+    opts: hard,bg
+    dump: 0
+    passno: 0
+    state: mounted
+    src: "{{item.src}}"
+    path: "{{item.path}}"
```

Add spaces around the variable names here to be consistent with convention and the remainder of our code:

```suggestion
    src: "{{ item.src }}"
    path: "{{ item.path }}"
```